tests/lightning_qubit/test_measurements_class.py

```diff
@@ -19,21 +19,28 @@
 import numpy as np
 import pennylane as qml
 import pytest
-from conftest import LightningDevice  # tested device
+from conftest import LightningDevice, device_name  # tested device
 from flaky import flaky
 from pennylane.devices import DefaultQubit
 from pennylane.measurements import VarianceMP
 from scipy.sparse import csr_matrix, random_array
 
-from pennylane_lightning.lightning_qubit import LightningQubit
+try:
+    from pennylane_lightning.lightning_qubit_ops import (
+        MeasurementsC64,
+        MeasurementsC128,
+    )
+except ImportError:
+    pass
+
 from pennylane_lightning.lightning_qubit._measurements import LightningMeasurements
 from pennylane_lightning.lightning_qubit._state_vector import LightningStateVector
 
-if not LightningQubit._CPP_BINARY_AVAILABLE:
-    pytest.skip("No binary module found. Skipping.", allow_module_level=True)
+if not LightningDevice._new_API:
```

Comment:
This initially skipped the test for all devices except `lightning.qubit`, but there is a device-dependent test in this file. (`TestMeasurements.test_probs_tape_unordered_wires`) which fails with the old device due to unordered wires being unsupported. So these tests only run with the new API now. Made the same change to `test_state_vector_class.py` and `test_adjoint_jacobian_class.py`.